rtloader/common/builtins/datadog_agent.c

```diff
@@ -455,7 +467,7 @@ static PyObject *set_external_tags(PyObject *self, PyObject *args)
         free(tags);
     }
 
-done:
+error:
```

Comment:
I like the old name "done" better, since we end up here even if there are no errors.